src/main/FetchService.curl.ts

```diff
@@ -29,6 +31,11 @@ export class FetchCurlService extends FetchService {
         ]);
         const { headers: responseHeaders, info } = this.parseStderr(stderr);
         const duration = Date.now() - startedAt;
+        this.metrics.serviceRequestLatency.addMillis(duration, {
```

Comment:
It should be just `requestLatency`, `fetch_service` in the metrics name refers to this repository.